monai/utils/enums.py

```diff
@@ -92,6 +94,21 @@ class NumpyPadMode(StrEnum):
     EMPTY = "empty"
 
 
+class NdimageMode(StrEnum):
+    """
```

Comment:
`NdimageMode` is not very explicit, I would suggest to add 1 line description for the usage of this class, for example: "Mode in the Ndimage to extend the input array beyond its boundaries."
What do you think?

Thanks.